build.gradle

```diff
@@ -5,10 +5,14 @@ buildscript {
     ext.support_libraries_version = '27.0.2'
     ext.espresso_version = '3.0.1'
     ext.kotlin_version = '1.2.30'
+    ext.spotbugs_version = '3.1.2'
 
     repositories {
         google()
         jcenter()
+        maven {
+            url "https://plugins.gradle.org/m2/"
```

Comment:
Oh, this library is not on jcenter?

Reply:
Is on jcenter but I forgot to cleanup the file. Sorry for that.